WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -741,14 +726,21 @@ private void addPreferencesForPreferenceScreen(PreferenceScreen preferenceScreen
         mTypePreferenceCategories.add(rootCategory);
     }
 
+    private void setPreferenceIcon(NotificationsSettingsDialogPreference preference, @DrawableRes int drawableRes) {
+        preference.setIcon(drawableRes);
+        preference.getIcon().setTintMode(Mode.SRC_IN);
+        preference.getIcon().setTintList(
+                ContextExtensionsKt
+                        .getColorStateListFromAttribute(preference.getContext(), R.attr.wpColorOnSurfaceMedium));
```

Comment:
I think indentation could be improved here as well, but it's a matter of preference _(no pun intended)_:

```
preference.getIcon().setTintList(ContextExtensionsKt
        .getColorStateListFromAttribute(preference.getContext(), R.attr.wpColorOnSurfaceMedium));
```